src/vcf/fixer.cpp

```diff
@@ -325,6 +487,30 @@ namespace ebi
             return columns.size() - written;
         }
 
+        size_t Fixer::replace_fields(const std::string &line,
+                                     const std::string &separators,
+                                     const std::string &expected_field,
+                                     std::function<bool(const std::string &column, size_t index)> condition_to_replace)
+        {
+            std::vector<std::string> columns;
+            util::string_split(line, separators.c_str(), columns);
+            size_t num_replaced_columns = 0;
+
+            for (size_t j = 0; j < columns.size(); ++j) {
+                if (not condition_to_replace(columns[j], j)) {
```

Comment:
I tend to find easier to understand the code when the positive condition is in the if branch, but it may be a personal preference.